packages/rendermime/src/mimemodel.ts

```diff
@@ -98,3 +68,61 @@ namespace MimeModel {
   }
 }
 
+
+/**
+ * A namespace for module private data.
+ */
+namespace Private {
+  /**
+   * The default implementation of an ibundle.
+   */
+  export
+  class Bundle implements RenderMime.IBundle {
```

Comment:
I would probably make this public as well, unless there is a good reason to not.